packages/cloud_functions/cloud_functions_web/test/cloud_functions_web_test.dart

```diff
@@ -23,9 +20,9 @@ void main() {
     final List<Map<String, dynamic>> log = <Map<String, dynamic>>[];
 
     Map<String, dynamic> loggingCall(
-        {@required String appName,
-        @required String functionName,
-        String region,
+        {required String? appName,
```

Comment:
required but nullable? Is this really desired?

Reply:
Missed this one during self review, it should be non-null have push a change